src/vs/workbench/contrib/welcome/gettingStarted/browser/gettingStarted.ts

```diff
@@ -1317,7 +1319,12 @@ export class GettingStartedPage extends EditorPane {
 		this.detailsScrollbar = this._register(new DomScrollableElement(stepsContainer, { className: 'steps-container' }));
 		const stepListComponent = this.detailsScrollbar.getDomNode();
 
-		reset(this.stepsContent, categoryDescriptorComponent, stepListComponent, this.stepMediaComponent);
+		const categoryFooter = $('.getting-started-footer');
+		// FIXME: This rendering doesn't actually work and is probably not right anwyays.
+		categoryFooter.append(renderFormattedText(localize('telemetryFooter',
```

Comment:
As you say rendering is broken, you need to render proper markdown.
This is what I currently see.

![Screenshot 2021-08-12 at 11 55 16](https://user-images.githubusercontent.com/1926584/129178580-db949cd7-f414-424a-b6c8-e387037acc82.png)

Reply:
Do you any recommendation on how to fix it, like which specific component to use for this kind of simpler paragraph with command-firing links?

Reply:
Fixed using MarkdownRenderer.